crates/rnote-ui/src/canvaswrapper.rs

```diff
@@ -601,6 +615,18 @@ mod imp {
                     }),
                 );
             }
+
+            {
+                // Context menu
+                self.touch_long_press_gesture.connect_pressed(
+                    clone!(@weak obj as canvaswrapper => move |_gesture, x, y| {
+                        let popover = canvaswrapper.contextmenu().popover();
+                        canvaswrapper.imp().last_contextmenu_pos.set(Some(na::vector![x, y]));
+                        popover.set_pointing_to(Some(&Rectangle::new(x as i32, y as i32, 0, 0)));
```

Comment:
use `gdk::Rectangle` here and remove the import of `Rectangle` at the start of the file